### PR TITLE
Misc cleanup surrounding Solidity contracts

### PR DESCRIPTION
**Pull request by awrichar**
Couple of standalone cleanup items identified while working #124. Might as well put them in while the larger development effort is still ongoing.

Depends on https://github.com/hyperledger-labs/firefly-cli/pull/66

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,9 +11,10 @@ RUN make build
 
 FROM node:14-alpine3.11 AS solidity-builder
 WORKDIR /firefly/solidity_firefly
-ADD solidity_firefly .
+ADD solidity_firefly/package*.json .
 RUN npm install
 RUN npm config set user 0
+ADD solidity_firefly .
```
Comment (nguyer) [inline Dockerfile:17]: I'm sure there's a good reason for what you did, but why do you only bring the `package.json` in first, and then everything else after that?

**Review by nguyer — APPROVED** [review]
One quick question. Otherwise looks good.

File: Dockerfile (hunk 1 above)
Comment (awrichar) [reply Dockerfile:17]: It allows Docker to cache the results of `npm install` (which only depends on package.json). That way if you make a small tweak to a Solidity file, you don't have to wait for Docker to re-download all the npm packages before compiling it.

Hint: I was making a lot of small tweaks to Solidity files (:

Comment (nguyer) [reply Dockerfile:17]: Ah yep. That totally makes sense. Good idea.

**Pull request merged**